source/scenes/Error/index.jsx

```diff
@@ -26,4 +29,9 @@ ErrorPage.propTypes = {
   message: PropTypes.string.isRequired,
 };
 
+function initializeReactGA() {
```

Comment:
This function is being defined, but it's never called anywhere. As a result, the code inside isn't actually executing anything.